<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 36, 36);
border-radius: 5px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QFrame" name="leftmenu_frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>51</width>
        <height>581</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>202</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(21, 21, 21);
border-radius: 10px;

									QScrollBar::vertical{
 										background: transparent;
                                        width: 14px;
                                        margin: 16px 2px 16px 2px;
                                        border-radius: 5px;}

                                 	QScrollBar::handle:vertical {
                                    	background-color: #4D545B;
                                        min-height: 30px;
                                        border-radius: 5px;}

QScrollBar::handle:vertical:hover {
                                    	background-color: #6A8AC8;
                                        /*&quot;border: 1px solid #6A8AC8;&quot;*/
                                        border-radius: 5px;}\n

QScrollBar::add-line::vertical{
                                    border:none;
									height: 10px;
                                    width: 10px;
                                    subcontrol-position: top;
                                    subcontrol-origin: margin;}
QScrollBar::sub-line::vertical{
                                    border:none;
									height: 10px;
                                    width: 10px;
                                    subcontrol-position: top;
                                    subcontrol-origin: margin;}
QScrollBar::add-page:vertical{
										border:none;
										background-color:none;}
QScrollBar::sub-page:vertical {
                                        background-color: none;
										border:none; }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="box_leftmenu">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="right_part_layout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QFrame" name="frame_titlebar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>46</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>571</width>
          <height>46</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(21, 21, 21);
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="title_layout" rowstretch="0" columnstretch="5,0,0,0,0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <property name="spacing">
            <number>5</number>
           </property>
           <item row="0" column="0">
            <widget class="QFrame" name="switcher_frame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>76</horstretch>
               <verstretch>36</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>76</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>76</width>
               <height>36</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius: 5px;
background-color: rgb(39, 39, 39);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="btn_deploy">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>36</horstretch>
               <verstretch>36</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QPushButton" name="btn_close">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>36</horstretch>
               <verstretch>36</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton::hover{
icon: url(&quot;:/resources/gui/icons/icon_close_hovered.svg&quot;);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="btn_minimise">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>36</horstretch>
               <verstretch>36</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_donate">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>36</horstretch>
               <verstretch>36</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>36</width>
               <height>36</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{background-color: rgb(39, 39, 39);
border-radius: 5px;
}
QPushButton::hover{
	background-color: rgb(30, 30, 30);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QFrame" name="frame_2">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="pages">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="minimumSize">
          <size>
           <width>574</width>
           <height>530</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="codepage_layout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
